src/Be.Vlaanderen.Basisregisters.Projector/Internal/Extensions/RetryPolicyConfigurationExtensions.cs

```diff
@@ -0,0 +1,41 @@
+namespace Be.Vlaanderen.Basisregisters.Projector.Internal.Extensions
+{
+    using System;
+    using ConnectedProjections;
+    using Exceptions;
+    using Microsoft.Extensions.Configuration;
+
+    internal static class RetryPolicyConfigurationExtensions
+    {
+        public static MessageHandlingRetryPolicy Configure(
```

Comment:
Likewise with my other comment, this is semi-generic with the Func, but then again we are expecting strategies which only take number of retries and a delay, which is specific again. 

Maybe better make this specific for now too till the need comes to extend it?